tests/harness/fake_tokenizer.py

```
@@ -51,6 +51,10 @@ def decode(self, token_ids, **kwargs) -> str:
         """Decode token IDs to string."""
         return TOKEN * len(token_ids)
 
+    def batch_decode(self, token_sequences, **kwargs) -> list[str]:
```

Ruff (0.15.2)

[warning] Unused method argument: `kwargs`

(ARG002)